src/components/Roadmap/RoadmapActionCard.tsx

```diff
@@ -87,8 +47,8 @@ const RoadmapActionCard: React.FC<IProps> = ({
       borderColor="lightBorder"
     >
       <Center background="cardGradient" h="260px">
-        <GatsbyImage
-          image={getImage(data[image])!}
+        <Image
+          src={imgSrc}
           alt={alt}
           objectFit="contain"
```

Comment:
`objectFit` prop is deprecated. We need to use the `style` prop from now on to specify that
```suggestion
          style={{
            objectFit: "contain",
          }}
```